tests/storage_test.go

```diff
@@ -71,15 +72,17 @@ var _ = Describe("Storage", func() {
 		var vmi *v1.VirtualMachineInstance
 		var nfsInitialized bool
 
-		initNFS := func() string {
+		initNFS := func(family k8sv1.IPFamily) string {
```

Comment:
can this and
https://github.com/kubevirt/kubevirt/pull/4156/files#diff-d642ef9101fdb0671d818d73a1c3f977R937
be combined ?